l10n_it_fatturapa/bindings/_ds.py

```diff
@@ -5,14 +5,14 @@
 # Namespace http://www.w3.org/2000/09/xmldsig# [xmlns:ds]
 
 from __future__ import unicode_literals
-import pyxb
-import pyxb.binding
-import pyxb.binding.saxer
+import pyxb  # pylint: disable=missing-import-error
```

Comment:
Anche qui (e negli altri file sotto), meglio usare `try..except` come indicato in https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#importerror

Inoltre, essendo codice autogenerato, queste modifiche sono da riportare nel README di questa cartella per far sì che vengano applicate anche quando verrà generato le prossime volte

Reply:
fatto